README.md

```diff
@@ -1,4 +1,38 @@
-"# spec-TemplateForSimplifierProjects"
+# About
+This template standardizes the development of fhir resources with .fsh resources. It should reduce time for setup and ease the development process.
```

Comment:
"It should reduce time for setup and ease the development process."
Das geht runter wie Butter. Sehr cool :)